### ISSUE TITLE
Create product landing page repo for CCE team

### ISSUE THREAD

**Issue by simonwex** [adobe/developer.adobe.com#130]

**Comment by simonwex** [comment]
I don't think this lives in adobedocs. This is content for the site that's not documentation-based. We might want to set up our own org for this, maybe not?

**Comment by filmaj** [comment]
We should discuss with @stevengill on this question.

**Comment by stevengill** [comment]
I don't think we should set up a new org for this. I'd be fine for this to live in adobedocs or the main adobe org. Being in the main adobe org means that we can use private repos for staging if we need to
